images/commons/entrypoint-readiness

```diff
@@ -1,19 +1,9 @@
-#! /bin/sh
+#!/bin/sh
 
-# simple readiness check to see if a late-running entrypoint has completed
-
-function version_check() {
-  test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1";
-}
+set -eo pipefail
```

Comment:
Hey @Schnitzel  I don't think `-o pipefail` is available in POSIX shell.

Reply:
mhh good question, but we use it here already: https://github.com/amazeeio/lagoon/blob/ad80d05a66f4b2030efe2ba75ee0661b1fb35bd4/images/commons/lagoon/entrypoints/999-readiness.sh#L3

Reply:
so we would need to remove it in both places?